stdlib/cmake/modules/SwiftSource.cmake

```diff
@@ -744,8 +744,6 @@ function(_compile_swift_files
         set(set_environment_args "${CMAKE_COMMAND}" "-E" "env" "DYLD_LIBRARY_PATH=${bs_lib_dir}")
       elseif(SWIFT_HOST_VARIANT_SDK MATCHES "LINUX|ANDROID|OPENBSD")
         set(set_environment_args "${CMAKE_COMMAND}" "-E" "env" "LD_LIBRARY_PATH=${bs_lib_dir}")
-      else()
-        message(FATAL_ERROR "TODO: bootstrapping support for ${SWIFT_HOST_VARIANT_SDK}")
```

Comment:
> dlls must be co-located or be found in Path.

That means you have to copy the stdlib dlls, or set the PATH, right?

Reply:
Correct.  But changing the `Path` is going to be very tricky here.  Fortunately, the dlls are emitted next to the binaries on Windows already :)